src/firefly/java/edu/caltech/ipac/visualize/plot/projection/TpvProjection.java

```diff
@@ -0,0 +1,321 @@
+package edu.caltech.ipac.visualize.plot.projection;
+
+import edu.caltech.ipac.visualize.plot.ProjectionException;
+import edu.caltech.ipac.visualize.plot.ProjectionPt;
+import edu.caltech.ipac.visualize.plot.Pt;
+
+
+/**
+ * See ref https://fits.gsfc.nasa.gov/registry/tpvwcs.html
+ * TPV distortion is applied to sky coordinates (compared to SIP/Gnomonic where distortion is applied to image pixel coordinates)
+ *
+ * This implementation was based on TPV concept and mathematics here (D. Shupe, et.al.):
+ *
+ * https://authors.library.caltech.edu/36801/
+ *
+ * In the paper above, intermediate coordinates (u,v) are renamed here fsamp,fline
+ * to follow same nomenclature as in @{@link GnomonicProjection}
+ * @author ejoliet
+ */
+public class TpvProjection {
+
+
+    static double dtr = Projection.dtr;
+    static double rtd = Projection.rtd;
```

Comment:
`dtr` and `rtd` should be final or it can be replaced by Projection.{dtr,rtd} respectively.  It was misleading, thinking values could changed.

Reply:
Good one.